test/common/quic/BUILD

```diff
@@ -281,3 +281,26 @@ envoy_cc_test_library(
         "@com_googlesource_quiche//:quic_test_tools_qpack_qpack_encoder_test_utils_lib",
     ],
 )
+
+envoy_cc_test(
```

Comment:
@alyssawilk I think you accidentally deleted this here: https://github.com/envoyproxy/envoy/pull/15540/files#diff-679677aa5d85fe778bee44c14d7a92b9d47b29f4a779557d7307f8ab39f074a6L12. Let me know if that's not the cae?

Reply:
deleted on purpose: I moved the test into  test/common/upstream/upstream_impl_test.cc to address the TODO


#ifdef ENVOY_ENABLE_QUIC
TEST_F(ClusterInfoImplTest, Http3) {

...